lib/shopify_api/resources/draft_order.rb

```diff
@@ -6,5 +6,9 @@ def send_invoice(draft_order_invoice = ShopifyAPI::DraftOrderInvoice.new)
       resource = post(:send_invoice, {}, draft_order_invoice.encode)
       ShopifyAPI::DraftOrderInvoice.new(ShopifyAPI::DraftOrder.format.decode(resource.body))
     end
+
+    def complete(options = {})
+      load_attributes_from_response(patch(:complete, {}, options.to_json))
```

Comment:
This should be `put`, since `patch` wasn't added until [ActiveResource 4.1.0](https://github.com/rails/activeresource/commit/55cfbee6f7671c966be9135e7a13f1ef1b0f710a) and we have support for AR 3.x too.